src/libnnstreamer-edge/nnstreamer-edge-mqtt-mosquitto.c

```diff
@@ -365,11 +365,21 @@ nns_edge_mqtt_subscribe (nns_edge_broker_h broker_h)
 }
 
 /**
- * @brief Get message from mqtt broker.
+ * @brief Get message from mqtt broker with infinite timeout.
```

Comment:
Suggestion: nns_edge_mqtt_get_message() is internal function for mqtt.
How about adding new param 'timeout' without new function?
```
int nns_edge_mqtt_get_message (nns_edge_broker_h broker_h, unsigned int timeout, void **msg, nns_size_t * msg_len);
```

Reply:
Thanks for the suggestion. Updated.